packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -728,7 +728,7 @@ export class MirrorNodeClient {
         }));
 
         if (searchableTypes.find(t => t === constants.TYPE_CONTRACT)) {
-            const contract = await this.getContract(entityIdentifier, requestId);
+            const contract = await this.getContract(entityIdentifier, requestId).catch(() =>  {return null;});
```

Comment:
This approach works, but do we think on promise catch for `resolveEntityType()` in `eth.ts` would satisfy all the cases?

Reply:
Another option, outside of this might be to rewrite this function utilizing more of a promise approach then we'd have a single catch.
This is good for now though

Reply:
Yeah, I thought and  tried bubbling  the  error and  it  works,  but  the issue  there is that when there are multiple  promises, then the  first  one  bubbles up  and  propagates  until  reaching  the  `server.ts` `logAndHandleResponse` returning  a  more  descriptive  error  message  to the user, but if  the  2nd  promise  were to throw exception as well,  the second one was being unhandled and thus also crashing,  that is why I decided to do this approach, also considering that we might need to circle back and do a refactor of the whole method.